lib/EntryManager/EntryWrapper.js

```diff
@@ -107,9 +107,24 @@ export default class EntryWrapper extends React.Component {
   onSave(entry) {
     const action = (entry.id) ? 'PUT' : 'POST';
     const rk = this.props.resourceKey ? this.props.resourceKey : 'entries';
+
     return this.props.parentMutator[rk][action](entry)
       .then(data => this.goTo(data))
-      .catch(error => this.showSubmitErrorCallout(error.message || error.statusText));
+      .catch(this.handleError);
+  }
+
+  handleError = async error => {
+    let errorMessage;
+
+    if (error.status === 422) {
+      const response = await error.json();
+      errorMessage = response.errors[0].message;
+    } else {
+      const { message, statusText } = error;
+      errorMessage = message || statusText;
+    }
+
+    this.showSubmitErrorCallout(errorMessage);
```

Comment:
This topic was already discussed with the team and Zak Burke. There no single pattern for implementation of translation of server-side error messages, and better to avoid one-off implementations. Also translation is out of scope this issue.

Reply:
Even if it is not a part of the issue, I think this good point to raise and discuss. Not necessarily to handle it in the scope of this issue though. Since these changes takes place as part of the shared component this why I raise it.
cc @zburke

Reply:
Agreed. I am glad you brought it up, and maybe if we keep beating the "It is wrong to forward localizable values from the server straight to the UI" drum, eventually we'll get some traction from a PO or a non-English institution. 

Here's the note I sent to @annamelnyk when she asked about this: 
> I would just forward the server side message directly to the UI. We do need to handle translation of server side error messages, but since we don’t have pattern for that implementation so we can be sure all modules function in the same way, I would like to avoid a one-off implementation. Does that make sense?
> 
> I wish we had a solution in place to manage translation of server-side error messages, but all the early-implementors have put a low priority on translation-related tasks. We have the same problem with permissions: their labels and descriptions are always displayed in English in ui-users. We have tickets for these problems ([UXPROD-370](https://issues.folio.org/browse/UXPROD-370), [UXPROD-371](https://issues.folio.org/browse/UXPROD-371)) but until a customer makes them a priority, the work will never get assigned to us developers even though it feels like an obvious bug  :shrug: